tests/test_medline_parser.py

```diff
@@ -1,9 +1,157 @@
 import os
 import pytest
+import requests
+import gzip
+from io import BytesIO
```

Comment:
Sort according to PEP8: Base packages in first block, 3rd party packages next block, own packages in last block. Within each block, full imports first (alphabetically), then partial imports (alphabetically)